shivanand.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+BASE_URL = "https://reqres.in/api/users"
+
+def apiCall():
```

Comment:
Method names must not be camel cases. Change them!! Also use type hinting necessary along with the return type.

Reply:
added